src/main/java/browserview/BrowserComponent.java

```diff
@@ -486,4 +486,9 @@ public String getCurrentUrl() {
         return driver.getCurrentUrl();
     }
 
+    public void switchToConversationTab() {
+        if (isCurrentUrlIssue() && GitHubURL.isPullRequestLoaded(getCurrentUrl())) {
```

Comment:
do we need the first check here? since it seems to be called too in all cases this method is called.